mvn_setup/tasks-parent/tasks-ui/src/core/TaskTable/CellMenu.tsx

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
-import { Box, MenuList, MenuItem, ListItemText, Divider } from '@mui/material';
+import { Box, MenuList, MenuItem, ListItemText, Divider, styled } from '@mui/material';
 import MoreHorizOutlinedIcon from '@mui/icons-material/MoreHorizOutlined';
-
+import DeleteIcon from '@mui/icons-material/Delete';
 import { FormattedMessage } from 'react-intl';
 
 import client from '@taskclient';
 import { usePopover } from './CellPopover';
 import CellHoverButton from './CellMenuButton';
 import TaskEditDialog from 'core/TaskEdit';
+import TaskWorkDialog from 'core/TaskWork';
 import { StyledTableCell } from './StyledTable';
 
+const StyledListItemText = styled(ListItemText)(({ theme }) => ({
```

Comment:
This style could be simplified just by making it a StyledBox and declaring the styles for `color`, `display`, and `alignItems`.  Then there is no need for span in here.